src/osp/Universe.cpp

```diff
@@ -75,4 +75,14 @@ Vector3 Universe::sat_calc_pos_meters(Satellite referenceFrame, Satellite target
     return Vector3(sat_calc_pos(referenceFrame, target)) / 1024.0f;
 }
 
+TypeSatIndex Universe::sat_type_find_index(std::string_view name)
+{
+    auto foundIt = m_typeSatIndices.find(name);
 
+    if (foundIt != m_typeSatIndices.end())
```

Comment:
recommend c++17 if-initializer

Reply:
like this?
```cpp
if (auto foundIt = m_typeSatIndices.find(name); foundIt != m_typeSatIndices.end())
```
I think it looks pretty weird. It feels like this hurts readability to save a line. I guess there might be some benefit if there was a long series of similar if statements, but this isn't the case here.

Reply:
Sure, either way.

Personally I prefer the consistency, and the reduction of variable scope, but it doesn't make a big difference.